packages/fxa-auth-server/lib/senders/emails/templates/inactiveAccountFirstWarning/en.ftl

```diff
@@ -0,0 +1,10 @@
+inactiveAccountFirstWarning-subject = Remember to confirm your account
+inactiveAccountFirstWarning-title = Do you want to keep your { -brand-mozilla } account and data?
```

Comment:
```suggestion
inactiveAccountFirstWarning-title = Do you want to keep your { -product-mozilla-account } and data?
```

Reply:
From my understanding `Mozilla account` does mean an individual's account so should we use `-product-mozilla-account` here?

Reply:
This one I wasn't sure about since Mozilla seemed to apply to both the account and the data, but maybe that's fine and localizers can work around it.

Reply:
I too was unsure. Yeah maybe what you have is better in terms of providing localizers the flexibility.